app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+
```

Comment:
remove blank line. You should put a blank line just  between different group of imports. Read PEP8 https://peps.python.org/pep-0008/#imports